.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+indent_style = space
+end_of_line = lf
+charset = utf-8
+insert_final_newline = true
+
+[*.{js,css}]
+indent_size = 2
+max_line_length = 80
+
+[*.html]
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = true
```

Comment:
Do we need this line?

Reply:
i think we do not need this as it was already in `[*]`